### PR TITLE
chore: rename "admin" and "site moderator" to moderator

### PR DESCRIPTION
**Pull request by luchaos**

### CODE REVIEW

File: resources/views/community/components/forum/post.blade.php
Comment (wescopeland) [inline resources/views/community/components/forum/post.blade.php:1]: nit: I think we are indenting the props over 4 spaces too far. I suspect this was an accidental side effect of increasing the indent within the root `@if()` directive.

**Current (this PR)**
```php
@if($isPreview || $canShowPost)
    <x-forum.post-container
            :commentId="$commentId ?? null"
            :isHighlighted="$isHighlighted ?? false"
            :isPreview="$isPreview ?? false"
    >
        <x-forum.post-author-box
                :authorUserName="$commentAuthor"
                :authorJoinDate="$commentAuthorJoinDate"
                :authorPermissions="$commentAuthorPermissions"
                :isAuthorDeleted="$commentAuthorDeletedDate !== null"
        />
```

**Proposed**
```
@if($isPreview || $canShowPost)
    <x-forum.post-container
        :commentId="$commentId ?? null"
        :isHighlighted="$isHighlighted ?? false"
        :isPreview="$isPreview ?? false"
    >
        <x-forum.post-author-box
            :authorUserName="$commentAuthor"
            :authorJoinDate="$commentAuthorJoinDate"
            :authorPermissions="$commentAuthorPermissions"
            :isAuthorDeleted="$commentAuthorDeletedDate !== null"
        />
```

Without this change, IMO the file is a bit more difficult to read.

**Comment by wescopeland** [comment]
Screenshot 2023-08-17 at 9 05 36 AM

Verified locally as functioning correctly.

File: resources/views/community/components/forum/post.blade.php [hunk 1]
```diff
@@ -32,7 +33,7 @@
     $commentDateModified = $commentData['DateModified'];
     $commentIsAuthorised = $commentData['Authorised'];
 
-    $isCurrentUserAdmin = $currentUserPermissions >= Permissions::Admin;
+    $isCurrentUserAdmin = $currentUserPermissions >= Permissions::Moderator;
```
Comment (wescopeland) [inline resources/views/community/components/forum/post.blade.php:36]: nit: Propose renaming this variable to `$isCurrentUserModerator`.

File: resources/views/community/components/forum/post.blade.php
Comment (luchaos) [reply resources/views/community/components/forum/post.blade.php:1]: Interesting, that was an automatic edit/formatting 🤔 
I agree, it shouldn't indent this far.
Fixed in https://github.com/RetroAchievements/RAWeb/pull/1772/commits/aecc9ba14dbecefab84b6a256e3e9eaeed420627

File: resources/views/community/components/forum/post.blade.php (hunk 1 above)
Comment (luchaos) [reply resources/views/community/components/forum/post.blade.php]: Changed in https://github.com/RetroAchievements/RAWeb/pull/1772/commits/aecc9ba14dbecefab84b6a256e3e9eaeed420627

**Review by wescopeland — APPROVED** [review]

**Pull request merged**